addons/point_of_sale/static/src/js/tests.js

```diff
@@ -1,15 +1,16 @@
 odoo.define('point_of_sale.Tour', function (require) {
     "use strict";
 
-    var Tour = require('web.Tour');
+    var tour = require("web_tour.tour");
 
     function add_product_to_order(product_name) {
         return [{
-            title: 'buy ' + product_name,
-            element: '.product-list .product-name:contains("' + product_name + '")',
+            content: 'buy ' + product_name,
+            trigger: '.product-list .product-name:contains("' + product_name + '")',
         }, {
-            title: 'the ' + product_name + ' have been added to the order',
-            waitFor: '.order .product-name:contains("' + product_name + '")',
+            content: 'the ' + product_name + ' have been added to the order',
+            trigger: '.order .product-name:contains("' + product_name + '")',
+            run: function () {}, // it's a check
```

Comment:
Yes but : 
Step 1 -> trigger found -> run click -> dom mutation -> Step 2 -> trigger found -> run empty (only a check) -> NO mutation -> ... the tour is blocked.
What am I missing? ^^